src/dune_rules/dir_contents.mli

```diff
@@ -40,8 +40,12 @@ val get : Super_context.t -> dir:Path.Build.t -> t Memo.Build.t
     not part of a group. *)
 val dirs : t -> t list
 
-type gen_rules_result =
-  | Standalone_or_root of t * t list  (** Sub-directories part of the group *)
+type triage =
+  | Standalone_or_root of
+      { root : t
+      ; subdirs : t list  (** Sub-directories part of the group *)
+      ; rules : Rules.t
+      }
   | Group_part of Path.Build.t
```

Comment:
I'm not sure what the "group part" name means. Perhaps, something more verbose like `Subdir_of_parent_group` would be clearer.

Reply:
It was meant to mean "part of a group". `Subdir_of_parent_group` looks good to me.